### PR TITLE
Implemented VT switching for KMSDRM on Linux

### PR DESCRIPTION
**Pull request by slouken**
Fixes https://github.com/libsdl-org/SDL/issues/3844

@vanfanel, @inolen, can you review this? I'm not sure what to do for Vulkan windows.

### CODE REVIEW

**Comment by vanfanel** [comment]
Looks good to me.
However, I don't know what to do in Vulkan either... It's been years since I did the Vulkan on TTY stuff.

**Comment by slouken** [comment]
Fair enough. Thanks!

File: src/core/linux/SDL_evdev_kbd.c [hunk 1]
```diff
@@ -296,6 +300,128 @@ static void kbd_register_emerg_cleanup(SDL_EVDEV_keyboard_state *kbd)
...
+};
+static int vt_release_signal;
+static int vt_acquire_signal;
+static SDL_AtomicInt vt_signal_pending;
+
+typedef void (*signal_handler)(int signum);
+
+static void kbd_vt_release_signal_action(int signum)
+{
+printf("kbd_vt_release_signal_action\n");
```
Comment (stsp) [inline src/core/linux/SDL_evdev_kbd.c:316]: Forgotten debug prints?

Comment (slouken) [reply src/core/linux/SDL_evdev_kbd.c:316]: Yep :)

**Comment by slouken** [comment]
Does Alt-fX work for you? It's been years since I've looked at this.

**Comment by stsp** [comment]
No, doesn't work either.
At the same time the app
see the keypresses properly,
as I added user to `input`,
so everything should work.

**Comment by slouken** [comment]
> No, doesn't work either. At the same time the app see the keypresses properly, as I added user to `input`, so everything should work.

Feel free to open a new issue for this, including your test environment so we can try to repro here.

**Pull request merged**